pcdsdevices/rs_powersupply.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+
+logger = logging.getLogger(__name__)
+
+
+class RSchannel(Device):
```

Comment:
I believe this should be capitalized, right? If so, make sure to change the other instances as well.
```suggestion
class RSChannel(Device):
```